src/sql/workbench/common/editor/query/queryPlan2State.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+
+export class QueryPlan2State {
```

Comment:
Class name is a little confusing at first glance. Is the "2" for queryPlanSecondState or queryPlanToState?

Reply:
It is the temp name for the QueryPlan2 state class. Once this feature replaces the original Query plan, I will rename it to QueryPlanState.